src/one_dragon/devtools/python_launcher.py

```
@@ -296,6 +296,23 @@ def fetch_latest_code(ctx: OneDragonEnvContext) -> None:
     else:
         print_message(f"{gt('代码更新失败')}: {msg}", "ERROR")
 
+
+def sync_dependencies(ctx: OneDragonEnvContext) -> None:
+    """使用 uv 同步运行依赖，并将原生进度输出透传到当前终端。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)